### PR TITLE
Data support / 1.8 item types

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: Lib/src/main/java/me/desht/dhutils/ItemNames.java [hunk 1]
```diff
@@ -95,16 +97,18 @@
...
+            .put("38:4", "Red Tulip")
+            .put("38:5", "Orange Tulip")
+            .put("38:6", "White Tulip")
+            .put("38:7", "Pink Tulip")
+            .put("38:8", "Oxeye Daisy")
             .put("39", "Brown Mushroom")
             .put("40", "Red Mushroom")
             .put("41", "Gold Block")
             .put("42", "Iron Block")
-            .put("43", "Stone Slab (Double)")
```
Comment (desht) [inline Lib/src/main/java/me/desht/dhutils/ItemNames.java:102]: Why are you removing these?  They can't be obtained normally, but they can certainly be obtained via plugins, so the descriptions need to stay.  There are some that need adding too.. 43:6 thru 43:9 also appear to be valid.

**Comment by ghost** [comment]
I removed it as it is being removed in the 1.8 update: http://prntscr.com/3y9k1f
Source: http://minecraft.gamepedia.com/Upcoming_features

Let me know if you'd like me to revert this change

**Comment by desht** [comment]
We'll need to keep those.  Since there isn't actually a 1.8 CB release at this point, we don't want to break 1.7 functionality.   And even after a 1.8 release, it's not critical to remove it immediately.

**Comment by ghost** [comment]
I see, well I have added the double slab items back for now then.

**Comment by desht** [comment]
Pulled, thanks!

**Pull request merged**